Zend/zend_virtual_cwd.c

```diff
@@ -1139,6 +1140,9 @@ CWD_API int virtual_file_ex(cwd_state *state, const char *path, verify_path_func
 	path_length = tsrm_realpath_r(resolved_path, start, path_length, &ll, &t, use_realpath, 0, NULL);
 
 	if (path_length == (size_t)-1) {
+#ifdef ZEND_WIN32
+		if (errno != EACCES)
+#endif
 		errno = ENOENT;
```

Comment:
This is a quick hack, but I wonder whether that wouldn't make sense on non Windows systems, too.